src/ScriptCs.Hosting/IVisualStudioSolution.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+using System.Collections.Generic;
+
+namespace ScriptCs.Contracts
+{
+    public interface IVisualStudioSolution
+    {
+        void AddHeader();
+        void AddScriptcsProject(string scriptcsPath, string workingPath, string args, bool attach, string projectGuid);
+        void AddProject(string path, string name, string guid, string[] files);
+        void AddGlobalHeader(string projectGuid);
+        void AddGlobalNestedProjects(IList<Tuple<string, string>> nestedItems);
```

Comment:
Ok
On Sun, Feb 14, 2016 at 4:41 AM Filip W <notifications@github.com> wrote:

> In src/ScriptCs.Hosting/IVisualStudioSolution.cs
> <https://github.com/scriptcs/scriptcs/pull/1109#discussion_r52842100>:
>
> > @@ -0,0 +1,15 @@
> > +using System;
> > +using System.Collections.Generic;
> > +
> > +namespace ScriptCs.Contracts
> > +{
> > +    public interface IVisualStudioSolution
> > +    {
> > +        void AddHeader();
> > +        void AddScriptcsProject(string scriptcsPath, string workingPath, string args, bool attach, string projectGuid);
> > +        void AddProject(string path, string name, string guid, string[] files);
> > +        void AddGlobalHeader(string projectGuid);
> > +        void AddGlobalNestedProjects(IList<Tuple<string, string>> nestedItems);
>
> can we get a better signature on a public member here? i.e. a descriptive
> type instead of a list of Tuple which is very difficult to comprehend
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/scriptcs/scriptcs/pull/1109/files#r52842100>.
>